charts/pvc-autoresizer/templates/controller/mutatingwebhookconfiguration.yaml

```diff
@@ -14,11 +14,13 @@ webhooks:
 - admissionReviewVersions:
   - v1
   clientConfig:
-    {{- with .Values.webhook.caBundle }}
-    caBundle: {{ . }}
+    {{- if .Values.webhook.caBundle }}
+    caBundle: {{ .Values.webhook.caBundle }}
+    {{- else if and (not .Values.webhook.existingCertManagerIssuer) (not (index .Values "cert-manager" "enabled")) }}
```

Comment:
Would you please reconsider this expression using new options? And `metadata.annotations."cert-manager.io/inject-ca-from"` too.